config-sample.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url": "vagrant.api.jquery.com",
-  "username": "admin",
-  "password": "secret"
+  "url": "local.api.jquery.com",
```

Comment:
We also need the scheme here, don't we?

Reply:
It actually works either way for me, but I did think at one point that was better for Andre. I'll add it here.